djstripe/utils.py

```diff
@@ -35,15 +37,22 @@ def subscriber_has_active_subscription(subscriber, plan=None):
         * user.is_superuser
         * user.is_staff
 
+    If price and plan are None and there exists only one subscription, this method will
+    check if that subscription is active. Calling this method with no price, no plan and
+    multiple subscriptions will throw an exception.
+
     :param subscriber: The subscriber for which to check for an active subscription.
     :type subscriber: dj-stripe subscriber
+    :param price: The price for which to check for an active subscription.
+    :type price: Price or string (price ID)
     :param plan: The plan for which to check for an active subscription.
-        If plan is None and there exists only one subscription, this method will
-        check if that subscription is active. Calling this method with no plan and
-        multiple subscriptions will throw an exception.
     :type plan: Plan or string (plan ID)
-
     """
+
+    if price and plan:
+        raise TypeError("price and plan arguments cannot both be defined.")
```

Comment:
Let's add a deprecationwarning if `plan` is defined at all (to switch to price).